draft-ietf-quic-recovery.md

```diff
@@ -693,22 +702,35 @@ follows:
    ssthresh = infinite
 ~~~
 
+### On Packet Sent
+
+Whenever a packet is sent, and is contains non-ACK frames,
```

Comment:
Instead of `and is contains non-ACK frames,` I assume you mean `and it contains non-ACK frames,`?